packages/internal/bridge/sdk/src/types/index.ts

```diff
@@ -124,7 +125,9 @@ export interface FeeData {
  * ensures the correct params are supplied when trying to calculate the fees
  */
 export type BridgeFeeRequest = DepositFeeRequest
+| DepositERC20FeeRequest
 | WithdrawFeeRequest
```

Comment:
Should we update WithdrawFeeRequest -> WithdrawNativeFeeRequest

and DepositFeeRequest -> DepositNativeFeeRequest

to be more descriptive.

Reply:
This sounds good, updated.